generator/osm2type.cpp

```diff
@@ -1394,6 +1394,10 @@ void GetNameAndType(OsmElement * p, FeatureBuilderParams & params,
           params.layer = base::Clamp(params.layer, int8_t{feature::LAYER_LOW}, int8_t{feature::LAYER_HIGH});
         }
       }},
+      {"is_sidepath:of:name", "*", [&params](string & k, string & v)
+      {
+        params.SetDefaultNameIfEmpty(std::move(v));
```

Comment:
Here the missing name of a sidewalk will be set from a nearby street, likely in the context of the recently discussed issue #8862

However, there could be issues with increased data size and with the search results. Also, only a few sidewalks have this tag set.

Would it be better to query nearby street names dynamically, only when needed?

Reply:
Yes, it should be implemented in another way. This (PR) approach is very straightforward and can be confusing in other use-cases like map rendering or search.

Reply:
agreed 👍️